lua/core/which-key.lua

```diff
@@ -207,6 +208,12 @@ M.setup = function()
 
   wk.register(mappings, opts)
   wk.register(vmappings, vopts)
+
+  if lvim.builtin.which_key.on_config_done then
+    lvim.builtin.which_key.on_config_done(wk)
```

Comment:
We have `which_key` and `wk`, can you remove `local wk = require "which-key"` and use which_key everywhere ?

Reply:
Done